CONTRIBUTING.md

```diff
@@ -138,8 +139,8 @@ New features:
 ### Devops
 New features:
 
-- If there are any new configuration properties, do the follow the same
-  structure as existing configuration style?
+- Does the configuration style follow the config guidelines? That is compile
+  (Maven) config goes in `pom.xml`, runtime (Spring) goes in `portal.properties`.
```

Comment:
I was thinking that any `pom.xml` would do, whichever is appropriate for the compile config. I guess if it applies only to `scripts/`. It should be in `scripts/pom.xml` etc. Will try to be more explicit there. It should just be clear that compile parameters should not be in neither `GlobalProperties.java` or `.properties`.

I assume `GlobalProperties.java` is only for accessing runtime config. If that's the case it could make sense to put the defaults in there. Then you can overload those parameters with a `.properties` file and ideally also environment variables. We also have Spring runtime config, so I'm not entirely sure how that fits in with the `GlobalProperties,java`.

Do you have any comments on this @morungos?